src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -237,10 +241,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 
 	func applicationDidBecomeActive(_ application: UIApplication) {
 		Log.info("Application did become active.", log: .appLifecycle)
-		
-		if CWAHibernationProvider.shared.isHibernationState {
-			applyEndOfLifeChanges()
-		}
```

Comment:
Did you checked, that there will be no side effect if we remove `applyEndOfLifeChanges()` in `applicationDidBecomeActive(:)`?